.gitsubprojects

```diff
@@ -1,5 +1,5 @@
 # -*- mode: cmake -*-
-git_subproject(ZeroEQ https://github.com/HBPVIS/ZeroEQ.git 1949f24)
+git_subproject(ZeroEQ https://github.com/HBPVIS/ZeroEQ.git ff94fa8)
```

Comment:
Why changed? Please revert, this would break the RestServer without the change for while(receive)